src/neo/SmartContract/Native/Oracle/OracleContract.cs

```diff
@@ -0,0 +1,114 @@
+#pragma warning disable IDE0051
+
+using Neo.Cryptography;
+using Neo.Cryptography.ECC;
+using Neo.Ledger;
+using Neo.Network.P2P.Payloads;
+using Neo.Persistence;
+using Neo.SmartContract.Manifest;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace Neo.SmartContract.Native.Oracle
+{
+    public sealed partial class OracleContract : NativeContract
```

Comment:
`OracleContract` -> `OracleContractAsync`, then we still have space for the other one to come, something like that.

Reply:
Maybe even `OracleContractAsyncFixed`, because the nodes are fixed and not open registration.
What we imagined in the beginning was that oracles would be open for any team to create their group and the `Sender` would leave default `oracles` or choose one from the registered ones in the `NativeContract`

Reply:
But looking at this template you created, @erikzhang, it looks simpler to adjust these points I mentioned and leave `sender` the option to pick an oracle group instead of the default ones.

Reply:
> then we still have space for the other one to come

We only need one native oracle.

> because the nodes are fixed and not open registration.

The nodes are selected by the committee.

Reply:
Oracle group is an interesting idea. I will think about it.

Reply:
I think oracle group is not necessary. We have the oracle nodes selected by the commitee. If you don't trust them, then you can build your own oracle contract. But few people trust private oracle contracts. Therefore, it is not worthwhile for the native oracle contract to support private groups.

Reply:
I agree with this perspective as well of private oracle contracts.
But it would be good to have a simple mechanism in the tx to make that simpler,perhaps list the hash of the private oracle contract optionally.